.github/workflows/deploy-multiple-accounts.yml

```diff
@@ -123,6 +123,28 @@ jobs:
         with:
           ssm_parameter: ${{env.TWILIO_SSM_PREFIX}}_POST_SURVEY_BOT_CHAT_URL
           env_variable_name: POST_SURVEY_BOT_CHAT_URL
+
+      # Since we can't provision a lex chat bot with e2e as the helpline code
+      # we will have to change the helpline code for the E2E account to "ee" on serverless
+      - name: Set Helpline Code for E2E Account
```

Comment:
We shouldn't have helpline specific exceptions in the core deploy workflow.

We should either:

- make this logic generic, like using the HELPLINE_CODE parameter if it is set for the helpline, or fall back to the existing logic if not

OR

- Add the _HELPLINE_CODE lookup specifically in a custom action for the E2E account - custom actions are run after the core setup so .env lines added there will override the core ones

I don't really have a preference for either